src/transformers/pipelines/automatic_speech_recognition.py

```diff
@@ -502,7 +502,7 @@ def _forward(self, model_inputs, return_timestamps=False, **generate_kwargs):
                         generate_kwargs["num_frames"] = num_frames
 
             tokens = self.model.generate(
-                inputs=inputs,
+                input_features=inputs,
```

Comment:
```suggestion
                inputs,
```

@ylacombe a proposal to handle the variable first kwarg name, in that case :sweat_smile: